mix.exs

```diff
@@ -36,7 +36,8 @@ defmodule DepsNix.MixProject do
       {:dialyxir, "~> 1.4", only: [:dev, :test], runtime: false},
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
       {:mint, "~> 1.0"},
-      {:stream_data, "~> 1.1", only: [:dev, :test]}
+      {:stream_data, "~> 1.1", only: [:dev, :test]},
+      {:ex_nar, "~> 0.3.0"}
```

Comment:
Minor nitpick: would you mind maintaining alphabetical order here?

Otherwise LGTM!